pcm_wav.cpp

```diff
@@ -45,10 +50,9 @@ PcmOutputWav::PcmOutputWav(const char *fn)
     header.nBitsPerSample = 16;
     strncpy(header.tagdata, "data", 4);
     header.datasize = 0;
-
     f = fopen(fn, "wb");
     if (f == NULL) {
-        perror("fopen");
+        perror("failed to open output file.");
```

Comment:
Do not put punctuation in the argument to perror (look at the docs to check what it does).